core/src/main/java/fj/data/List.java

```diff
@@ -1497,7 +1546,7 @@ private void tail(final List<A> tail) {
    * @return A list of the given value replicated the given number of times.
    */
   public static <A> List<A> replicate(final int n, final A a) {
-    return n <= 0 ? List.<A>nil() : replicate(n - 1, a).cons(a);
+    return n <= 0 ? List.<A>nil() : List.<A>replicate(n - 1, a).cons(a);
   }
 
```

Comment:
Tried replicate on my Windows box, Java 1.8.0-25, Intellij 14.0.2 and no complaint.  I suggest you grab the latest community edition of Intellij, I think 14.0.3 just got released.